src/components/radio/radio_button.html

```diff
@@ -0,0 +1,22 @@
+<!-- TODO(jelbourn): render the radio on either side of the content -->
+
+<!-- The actual `radio` part of the control. -->
+<div class="md-radio-container">
+  <div class="md-radio-outer-circle"></div>
+  <div class="md-radio-inner-circle"></div>
+  <div class="md-ink-ripple"></div>
+</div>
+
+<input #input class="md-radio-input" type="radio"
```

Comment:
Can you add a TODO that's something like
"Evaluate trade-offs of using native radio vs. the cost of additional bindings"